### ISSUE TITLE
[Frontend Req] Loading Component before the first block loads.

### ISSUE THREAD

**Issue by abheektripathy** [availproject/light-client-web#27]
We can have a loading component or ghost ui until the first block loads so the page doesn't look empty.
Currently it looks like this before we see the first block.

<img width="1240" alt="Screenshot 2023-12-19 at 8 10 39 PM" src="https://github.com/availproject/light-client-web/assets/90976669/9f1e9c0d-6325-42bc-a930-e601a2b80b48">

**Comment by Megumiiiiii** [comment]
I can do this

![Screenshot_548](https://github.com/availproject/light-client-web/assets/98658943/427f2a37-a178-4adc-af34-587884e8e5b6)

**Comment by abheektripathy** [comment]
i was thinking more on the perspective of having a skeleton ui until the first block loads, lemme spin something up on figma.

**Comment by Megumiiiiii** [comment]
yeah it's you can check it here https://avail.light.kessoku.xyz/ , i think cube is more cooler so edit that to cube

**Comment by discord239** [comment]
Hi @Megumiiiiii, There is some issue with the night mode toggle switch, On first click the toggle icon doesn't switch but the UI switches. Kindly check.

**Comment by Megumiiiiii** [comment]
@discord239 edited to 1 icon only also moving the icon to the top right on mobile view
https://github.com/availproject/light-client-web/pull/26/commits/99269929a776aa9bd8be6541594cfccea38932ed

**Comment by cpp-phoenix** [comment]
@abheektripathy Is it under development. Otherwise I can take it up

**Comment by cpp-phoenix** [comment]
Hey @Megumiiiiii, Is it closed or still open?

**Comment by Megumiiiiii** [comment]
i think they gone.. they've been ignoring every PRs and issues for weeks

**Comment by abheektripathy** [comment]
what we are looking for here is until the first block loads, the whole page should have a loading animation of sorts, getting the animation ready, then we can implement the loading logic.

**Comment by cpp-phoenix** [comment]
I can work on it if @Megumiiiiii is not working on it anymore

**Comment by Megumiiiiii** [comment]
@cpp-phoenix take it

**Comment by abheektripathy** [comment]
perf, @cpp-phoenix let me get a animation from the team, then let's work on implementing it.
